README.md

```diff
@@ -47,24 +47,39 @@ SIC_CODE_API_PORT                    | Application Port
 SIC_CODE_API_MONGO_URL               | URL to MongoDB                                                            | `mongodb://mongo:27017`
 SIC_CODE_API_DATABASE                | Name of Sic Code Mongo database                                           | sic_code
 
-## Why a free text search was used rather than a regular expression find
+## What is MongoDB full text search and why do we used it over a regular expression find
 
-The main advantage of a free text search is that you can order the results in order of relevance (this would need to be implemented manually with a regular expression file)
+MongoDB full text search allows us to search string content in our collections and return matches based on the search term we have entered. It allows us to return matched values exactly or values that partially match a phrase.
```

Comment:
Instead of "partially match a phrase" say "match at least one word within the phase (a word match)."